packages/core/src/lib/uikitExcludePattern.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
+var _ = require('lodash');
+
 const uikitExcludePattern = (pattern, uikit) => {
   const state = pattern.patternState;
-  return uikit.excludedPatternStates.includes(state);
+  const tags = pattern.tags;
+
+  return uikit.excludedPatternStates.includes(state) || _.intersection(uikit.excludedTags, tags).length > 0;
```

Comment:
`uikit.excludedPatternStates` is empty in one test, the same could happen with a `patternlab-config`. If we use lodash instead, the error will be catched and no `undifined` will occur.

```suggestion
  return (
    _.includes(uikit.excludedPatternStates, state) ||
    _.intersection(uikit.excludedTags, tags).length > 0
  );
```